.pre-commit-config.yaml

```diff
@@ -51,3 +55,9 @@ repos:
     - id: no-commit-to-branch
     - id: trailing-whitespace
       exclude: ^vendor
+
+- repo: https://github.com/openstack-dev/bashate.git
```

Comment:
I noticed this was changed to `openstack` instead of `openstack-dev` in the similar Cinder PR.  Should it be changed here as well?

Reply:
Yes. you are right.. I've missed it. Changing. Thanks!